pom.xml

```diff
@@ -630,6 +630,7 @@
         <module>cpu_features</module>
         <module>modsecurity</module>
         <module>systems</module>
+        <module>leapmotion</module>
```

Comment:
Leapmotion should also be registered as an available module for the platforms you're building. Each JavaCPP platform has its own profile in this pom and leapmotion should be registered in these.